forest/geo.py

```diff
@@ -183,10 +193,23 @@ def to_180(x):
     return y
 
 
+def extend_180(x, lons):
```

Comment:
This function edits a copy of x and returns the copy. But our calling code uses "in-place" syntax, e.g. it doesn't assign to anything, `extend_180(gx, lons)`.